docs/api/hardhat/testing.md

```diff
@@ -60,14 +65,15 @@ The full list of the addresses of these accounts with the corresponding private
 
 The initial version of the local node was shipped with several ERC-20 tokens deployed by default.
 
-It's no longer the case and if you need to interact with ERC-20 tokens, you should deploy them yourself.
+That's no longer the case and **if you need to interact with ERC-20 tokens, you should deploy them yourself.**
```

Comment:
Should we maybe just remove the info about erc20s? Or move to a last section of troubleshoot or decpracted things? Just so we don't add an info that is only valid for a small set of people given the large Dev base will be new

Reply:
I think it's important to mention ERC20s, even more if this was a feature in the past.  I've simplified the message and leave it as a warning in the rich wallets section. I think that's the most appropriate place to have it for now

Reply:
fixed in e28f4343571352dec4b23cfb303616a2f8f497fa